Update moment dependency

himdel opened the pull request:
Depending on ~2.10.3 only works with moment 2.10.\*,
the current version is 2.12 .. AFAICT it seems moment is semver enough for us to just say 2.\* is OK.

@benjaminoakes do you agree? Would you mind doing a new release please, if so?

philfreo commented:
@benjaminoakes Can this be merged?

benjaminoakes commented:
I'd be okay with merging and prepping a release, but it's worth mentioning that I don't currently have a project that uses moment.js. I'd certainly appreciate any time anyone is willing volunteer on this.  I'm going to leave a couple comments on the PR.

benjaminoakes commented on bower.json:
```diff
@@ -21,6 +21,6 @@
     ],
     "dependencies": {
-        "moment": "~2.10.3"
+        "moment": "~2.12"
```
I like the idea of making this `2.*` given the semver discussion.  Let's do that.

benjaminoakes replied:
We should do the same change to `package.json`.

benjaminoakes replied:
I have fairly little experience with Bower and would appreciate assistance in confirming that any updates work correctly.  (Some simple script I could run would be great!)

benjaminoakes replied:
On this note, if it's not necessary to have the `build` directory any longer (with the minified JS), I'd like to drop it.  Bower expertise would be appreciated on this question.

himdel replied:
@benjaminoakes Note that you can't *just* drop `build/` for bower, since it's the only version of the JS that gets into bower right now. But if you remove `lib/` from the `ignore` section in `bower.json`, and if you're intending to keep this ES5 only, then sure.. :)

I'll update the PR with the requested changes today..

himdel commented:
@benjaminoakes I've updated the PR to change both the bower and the npm dependency to `~2.10`.

That means the same as `2.*` but only for `>= 2.10.0`. (Because we probably should not allow versions lower than 2.10 if that's the lowest one we've been depending on until now.)

I've also removed `lib/` from the bower's ignore list, so feel free to drop `build/` before the next release  ;).

Thanks..

benjaminoakes commented:
Thanks for your assistance!

benjaminoakes commented:
Released in v0.2.0.

As I mentioned, I no longer have a project that's actively using moment.js, so I would appreciate your feedback on upgrading.

himdel commented:
@benjaminoakes Thanks, it looks like 0.2.0 works just right for us :)

benjaminoakes commented:
@himdel Glad to hear it. :smile:  I'm happy to see ManageIQ is using it.  Out of curiosity, are you a developer with RedHat or a 3rd party contributor to ManageIQ?

himdel commented:
@benjaminoakes Hey, glad to know you've heard about ManageIQ :) And yup, I'm a RedHatter.. :)
